packages/e2e-tests/examples.json

```diff
@@ -41,5 +41,6 @@
   "cypress/doc_inspector_styles": "aa6201d2-1e83-408b-8247-cf3d26ac421a",
   "doc_minified_chromium.html": "5cdb2ffe-beff-4d35-9689-9242bd277f2d",
   "cypress-realworld/bankaccounts.spec.js": "6d4abbe1-2fcb-47d9-92d5-f7b45760b01a",
-  "flake/adding-spec.ts": "7b0db00a-d9a9-4d2d-b816-af14a1b682a6"
+  "flake/adding-spec.ts": "7b0db00a-d9a9-4d2d-b816-af14a1b682a6",
+  "redux/dist/index.html": "cd5327bc-5eed-4673-93bc-11e361f822da"
```

Comment:
@Mokshit06 we need to clone this recording in dev environment for backend E2E. I can help you with that. Ping me when you're wrapping up this PR.